jwt/token.js

```diff
@@ -0,0 +1,7 @@
+import jwt from "jsonwebtoken";
+
+export const generateToken = (user) => {
+  const signature = 'my-super-secret';
+  const expiration = '6h';

```

Comment:
There is little sense declaring a variable with the value you gonna use one time - especially when they are being passed into named dictionary. But be careful about magic numbers.